src/db/search/helpers/does-match.ts

```diff
@@ -0,0 +1,18 @@
+import { flatten } from "lodash";
+import { stripStringOfAllNonAlphaCharacters } from "./strip-characters";
+
+export function doesMatch(data: { input: string }, ...dataToQuery: string[]) {
+  const flattenedData = flatten(dataToQuery.map((element) => {
+    return stripStringOfAllNonAlphaCharacters(element).split(" ");
+  }));
+
+  const stringWithAllCharactersRemoved = stripStringOfAllNonAlphaCharacters(data.input);
+
+  const query = stringWithAllCharactersRemoved.toLowerCase().trim().split(" ");
+  for (let i = 0; i < flattenedData.length; i++) {
+    if (query.indexOf(flattenedData[i].toLowerCase()) >= 0 ) {
+      return true;
+    }
+  }
+  return false;
+}

```

Comment:
not sure we'll need these functions

Reply:
Maybe comment them out until they're used or removed... that way if we forget about them we won't go searching for where they're used

Reply:
I deleted these functions.